src/wakefs/namespace.cpp

```diff
@@ -193,50 +192,60 @@ static bool do_squashfuse_mount(const std::string &source, const std::string &mo
 
   int err = mkdir_with_parents(mountpoint, 0555);
   if (0 != err) {
-    std::cerr << "mkdir_with_parents ('" << mountpoint << "'):" << strerror(err) << std::endl;
+    std::cerr << "mkdir_with_parents ('" << mountpoint << "'): " << strerror(err) << std::endl;
+    return false;
+  }
+
+  auto fifo_path_result = mktempfifo();
+  if (!fifo_path_result) {
+    std::cerr << "mktempfifo ('" << strerror(fifo_path_result.error()) << "'): " << std::endl;
     return false;
   }
 
   pid_t pid = fork();
   if (pid == 0) {
     // kernel to send SIGKILL to squashfuse when wakebox terminates
     if (prctl(PR_SET_PDEATHSIG, SIGKILL) == -1) {
-      std::cerr << "squashfuse prctl: " << strerror(errno) << std::endl;
+      std::cerr << "squashfuse_ll prctl: " << strerror(errno) << std::endl;
+      unlink(fifo_path_result->c_str());
       exit(1);
     }
-    execlp("squashfuse", "squashfuse", "-f", source.c_str(), mountpoint.c_str(), NULL);
+    execlp("squashfuse_ll", "squashfuse_ll", "-o", "notify_pipe", fifo_path_result->c_str(), "-f",
```

Comment:
DIdn't we  call this `wake_squashfuse_ll` not `squashfuse_ll`

Reply:
Yes thanks for catching that. I guess I'll open another PR to address these issues.